src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+import duke.commands.Command;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Represents the initialization of Duke application.
+ * Initializes the application and starts the interaction with the user.
+ */
+public class Duke {
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * @param filePath to read/write task list data
+     * @throws IOException
+     */
```

Comment:
There are some violations of the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments) with respect to the Javadoc form.